scripts/build-cursors

```diff
@@ -67,3 +69,27 @@ while read ALIAS ; do
 
 	ln -s "$TO" "$OUTPUT/cursors_scalable/$FROM"
 done < $ALIASES
+
+while read ALIAS ; do
+	FROM=${ALIAS% *}
+	TO=${ALIAS#* }
+
+	if [[ -e "$HL_DIR/$FROM" ]]; then
+		continue
+	fi
+
+	if [ -f "$HL_DIR/$TO/meta.hl" ]; then # need to ensure $TO is not a mapping itself
+    echo -ne "\ndefine_override = $FROM" >> $HL_DIR/$TO/meta.hl;
+  else
+    grep=$(grep -m 1 "^$TO " $ALIASES)
+    TO=${grep#* }
+    echo -ne "\ndefine_override = $FROM" >> $HL_DIR/$TO/meta.hl
+  fi
+done < $ALIASES
```

Comment:
Please make sure your formatting follows the `.editorconfig`:

https://github.com/catppuccin/cursors/blob/42d57f790c0bac1c9948063fb8b548a67adfe2af/.editorconfig#L7-L13

At the moment, this is mixed in with tabs and spaces and I think some indentations are 4 instead of 2.